src/components/element-editor/element-editor.styles.tsx

```diff
@@ -1,8 +1,14 @@
 import { Accordion } from "@lifesg/react-design-system/accordion";
+import { Alert } from "@lifesg/react-design-system/alert";
 import styled from "styled-components";
 
 export const AccordionWrapper = styled(Accordion)`
+    margin-top: 1rem;
     & > div:first-child {
         display: none;
     }
 `;
+
+export const SaveChangesAlert = styled(Alert)`
+    margin: 1rem;
+`;
```

Comment:
layout does not match what is on figma:
- alert box should be shorter than accordion wrapper
- alert box should have vertical space of 1 rem

Reply:
there should be a 1rem spacing to the right of the arrow

<img width="303" alt="image" src="https://github.com/LifeSG/web-form-builder/assets/60728851/017e9abb-a969-474c-8f77-f43985224f39">
<img width="203" alt="image" src="https://github.com/LifeSG/web-form-builder/assets/60728851/1acb3c95-d159-4f62-9943-d028b1d77204">